consumer/src/main/java/com/ihren/consumer/PersonConsumer.java

```diff
@@ -0,0 +1,14 @@
+package com.ihren.consumer;
+
+import com.ihren.model.Person;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+import java.util.function.Consumer;
+
+@Component
+public class PersonConsumer {
+    @Bean
```

Comment:
If you use @Bean, @Configuration annotation should be used instead if @Component